index.html

```diff
@@ -3675,6 +3677,14 @@ <h3>Data Validation</h3>
 											href="https://infra.spec.whatwg.org/#list-remove">remove</a>
```

Comment:
In step (2), it should be <var>data["uniqueResources"]</var>, and not <var>["uniqueLinks"]</var>